sources/engine/Stride.Shaders/Compiler/EffectCompilerCache.cs

```diff
@@ -70,6 +70,9 @@ public override TaskOrResult<EffectBytecodeCompilerResult> Compile(ShaderMixinSo
             var compiledUrl = string.Format("{0}/{1}", CompiledShadersKey, mixinObjectId);
 
             var bytecode = new KeyValuePair<EffectBytecode, EffectBytecodeCacheLoadSource>(null, EffectBytecodeCacheLoadSource.JustCompiled);
+
+            goto label_force_compile;
+
```

Comment:
Will have to remove that before merge, just something to keep in mind

Reply:
Roger!